src/components/DeleteKebab/DeleteKebab.tsx

```diff
@@ -48,6 +52,17 @@ const DeleteKebab = ({
           : 'Remove selected repositories'}
       </DropdownItem>
     </ConditionalTooltip>,
+
+    <ConditionalTooltip
+      key='snapshot'
+      content='Manually trigger a snapshot in the repository'
+      show={true}
```

Comment:
We'd likely want to hide this dropdownItem if the user does not have rbac.write access, we also need to check the "features" from the AppContext to see if snapshotting is enabled.